stewart/theme.json

```diff
@@ -0,0 +1,361 @@
+					"name": "Background"
+				},
+				{
+					"slug": "primary",
+					"color": "#000000",
+					"name": "Primary"
+				},
+				{
+					"slug": "secondary",
+					"color": "#863119",
+					"name": "Secondary"
+				},
+				{
+					"slug": "tertiary",
+					"color": "#FFFFFF",
+					"name": "Tertiary"
+				}
+			]
+		},
+		"custom": {
+			"body": {
+				"typography": {
+					"lineHeight": 1.6
+				}
+			},
+			"heading": {
+				"typography": {
+					"lineHeight": 1.2
+				}
+			},
+			"margin": {
+				"horizontal": "30px",
+				"vertical": "30px"
+			}
+		},
+		"layout": {
+			"contentSize": "826px",
+			"wideSize": "1160px"
+		},
+		"typography": {
+			"fontFamilies": [
+				{
+					"fontFamily": "'Work Sans', -apple-system, BlinkMacSystemFont, 'Helvetica Neue', 'Helvetica', sans-serif",
+					"slug": "work-sans",
+					"name": "Work Sans"
+				}
+			],
+			"fontSizes": [
+				{
+					"name": "Tiny",
+					"size": "16px",
+					"slug": "tiny"
+				},
+				{
+					"name": "Small",
+					"size": "18px",
+					"slug": "small"
+				},
+				{
+					"name": "Normal",
+					"size": "20px",
+					"slug": "normal"
+				},
+				{
+					"name": "Large",
+					"size": "24px",
+					"slug": "large"
+				},
+				{
+					"name": "Extra Large",
+					"size": "32px",
+					"slug": "extra-large"
+				},
+				{
+					"name": "Huge",
+					"size": "38px",
+					"slug": "huge"
+				},
+				{
+					"name": "Gigantic",
+					"size": "50px",
+					"slug": "gigantic"
+				}
+			]
+		}
+	},
+	"styles": {
+		"blocks": {
+			"core/button": {
+				"border": {
+					"radius": "0"
+				},
+				"color": {
+					"background": "var(--wp--preset--color--foreground)",
+					"text": "var(--wp--preset--color--background)"
+				},
+				"typography": {
+					"fontSize": "var(--wp--preset--typography--font-size--normal)"
+				}
+			},
+			"core/code": {
+				"spacing": {
+					"padding": {
+						"left": "var(--wp--custom--margin--horizontal)",
+						"right": "var(--wp--custom--margin--horizontal)",
+						"top": "var(--wp--custom--margin--vertical)",
+						"bottom": "var(--wp--custom--margin--vertical)"
+					}
+				},
+				"border": {
+					"color": "#CCCCCC",
+					"radius": "0px",
+					"style": "solid",
+					"width": "2px"
+				}
+			},
+			"core/post-featured-image": {
+				"spacing": {
+					"margin": {
+						"top": "0",
+						"bottom": "var(--wp--custom--margin--horizontal)"
+					}
+				}
+			},
+			"core/navigation": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--small)",
+					"lineHeight": 2
+				}
+			},
+			"core/post-title": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
+					"fontSize": "var(--wp--preset--font-size--huge)",
+					"lineHeight": "var(--wp--custom--heading--typography--line-height)"
+				},
+				"spacing": {
+					"margin": {
+						"top": "0",
+						"bottom": "var(--wp--custom--margin--horizontal)"
+					}
+				}
+			},
+			"core/post-date": {
+				"color": {
```

Comment:
Setting these colors means that when a user updates the colors in the site editor, these ones stay the same.